src/signal-handling.c

```diff
@@ -53,13 +53,6 @@ JL_DLLEXPORT void jl_exit_on_sigint(int on)
     exit_on_sigint = on;
 }
 
-// what to do on SIGINT
-JL_DLLEXPORT void jl_sigint_action(void)
```

Comment:
we eventually need to go through and remove all of the non-exported API stuff from there. I just keep doing a (very) little at a time (like here)